filter/copy_fields.go

```diff
@@ -0,0 +1,75 @@
+package filter
+
+import (
+	"errors"
+	"fmt"
+	"sync/atomic"
+
+	"github.com/AdRoll/baker"
+)
+
+var CopyFieldsDesc = baker.FilterDesc{
+	Name:   "CopyFields",
+	New:    NewCopyFields,
+	Config: &CopyFieldsConfig{},
+	Help:   `Copy a field value to another field. Can copy multiple fields.`,
+}
+
+type CopyFieldsConfig struct {
+	FieldsMap []string `help:"List of fields to replace, as: [\"sourceField1\", \"targetField1\", \"sourceField2\", \"targetField2\"]" required:"true"`
```

Comment:
I'm not sure about calling this `FieldsMap`, since it implies a map datatype somehow. `FieldPairs` or just `Fields` what do you think?

Reply:
renamed to `CopyFields` and `ReplaceFields`